pkg/sensors/tracing/kprobe_copyfd_test.go

```diff
@@ -50,22 +43,15 @@ func TestCopyFd(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-
-	go func() {
-		logOut(t, "stdout> ", testPipes.StdoutRd)
-	}()
-
-	go func() {
-		logOut(t, "stderr> ", testPipes.StderrRd)
-	}()
+	defer testPipes.Close()
 
 	// makeSpecFile creates a new spec file bsed on the template, and the provided arguments
 	makeSpecFile := func(pid string) string {
 		data := map[string]string{
 			"MatchedPID": pid,
 			// NB: if this is a kata run, the pid that the ns-tester will print,
 			// will be inside the namespace.
-			"NamespacePID": fmt.Sprintf("%t", kataRun),
+			"NamespacePID": "false",
```

Comment:
Since kata is gone, can't we just fix up the templates to remove this altogether?

Reply:
That's a good point. Let's do it as a followup though. I did, however, removed the comment referring to the kata run.